docs/content/docs/start/tutorial.md

```diff
@@ -59,21 +59,20 @@ Fluid data is stored within containers, and these containers need to be created
 
 ### Create a new container
 
-The creation section of the application starts with calling `createContainer` and passing in a schema defining which shared objects will be available on the new `container`. After a new container is created, default data can be set on the shared objects before the container is attached to the Tinylicious service.
+The creation section of the application starts with calling `createContainer` and passing in a schema defining which shared objects will be available on the new `container`. After a new container is created, default data can be set on the shared objects before the container is attached to the Tinylicious service. Note, we are passing the parameter "2" in the createContainer call to indicate the version of Fluid Framework that the data
+is compatible with. For new apps always use "2". Use "1" if you have apps running FluidFramework 1 until you've migrated all the apps to version 2.
```

Comment:
I'm a bit iffy with the wording here but don't know how I'd write it. I would think of each app as an independent thing, unless more than 1 app shares the same containers. If they are independent, I can have app 1 using its 1.x containers and that should not make me use compatibility "1" on a different app. But for any given app, if I expect to have _clients_ running FF 1.x, then that's probably when I want to use compatibility "1", until all clients are using FF 2.0 and the app has been updated as necessary to support it?
```suggestion
is compatible with. For new apps always use "2". Use "1" if you have apps running FluidFramework 1.x until you've migrated all the apps to version 2.
```